index.html

```diff
@@ -10,346 +10,305 @@
-                      <a href="https://github.com/dagmawiii03"
-                        ><i class="fa-brands fa-github"></i
-                      ></a>
-                    </li>
-                    <li>
-                      <a href=""><i class="fa-solid fa-envelope"></i></a>
-                    </li>
-                  </ul>
-                  <p class="grey">dagmawi.yoha03@gmail.com</p>
-                  <br />
-                  <br />
-                </section>
-              </section>
-            </section>
-          </footer> 
-        </main>
-</body>
-</html>
\ No newline at end of file
+          </article>
+        </li>
+      </ul>
+    </section>
+    <footer class="footer-container">
+      <section class="footer-form">
+        <h1 class="footer-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h1>
+        <form action="https://formspree.io/f/xoqzdylg" method="post">
+          <ul class="form-fields">
+            <li class="input-name">
+              <input
+                type="text"
+                name="name"
+                maxlength="30"
+                placeholder="Full name"
+                required
+              />
+            </li>
+            <li class="input-email-form">
+              <input
+                class="input-email"
+                type="email"
+                name="email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                name="text_area"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                placeholder="Enter text here"
+                required
+              ></textarea>
+            </li>
+            <li class="submit-button">
+              <button type="submit">Get in touch</button><br />
+              <small></small>
+            </li>
+          </ul>
+        </form>
+      </section>
+      <ul class="social-media">
+        <li>
+          <a href="">
+            <img src="./assets/social-media/twitter.svg" alt="twitter icon" />
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com">
+            <img src="./assets/social-media/linkedin.svg" alt="linkedin icon" />
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <img src="./assets/social-media/medium.svg" alt="medium icon" />
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <img src="./assets/social-media/github.svg" alt="github icon" />
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <img
+              src="./assets/social-media/angellist.svg"
+              alt="angellist icon"
+            />
+          </a>
+        </li>
+      </ul>
+      <a href="www.gmail.com" class="link-mail">dagmawi.yoha03@gmail.com</a>
```

Comment:
Please hide the mail adress on the desktop view like on the Figma template.
<img width="755" alt="Screenshot 2023-02-26 at 12 00 02 AM" src="https://user-images.githubusercontent.com/25887253/221380314-898e2ad9-e9c4-4e69-b5fd-8bab0e36c9ec.png">
<img width="755" alt="Screenshot 2023-02-26 at 12 00 12 AM" src="https://user-images.githubusercontent.com/25887253/221380315-79805efa-a4f4-4363-ab26-d4cc83377418.png">

Reply:
Done